### ISSUE TITLE
let user config hyper-parameter sampling seed

### ISSUE THREAD

**Issue by smilesun** [marrlab/DomainLab#157]

**Comment by schoersch** [comment]
The seed for hyperparameter sampling is set via `sampling_seed` in the config yaml.

Please reopen if you meant something different.
